third_party/move/tools/move-coverage/src/source_coverage.rs

```diff
@@ -38,22 +38,84 @@ pub struct FunctionSourceCoverage {
     pub uncovered_locations: Vec<Loc>,
 }
 
+/// Source-level positive coverage information for a function.
+#[derive(Clone, Debug, Serialize)]
+pub struct FunctionSourceCoverageV2 {
```

Comment:
Could you add a comment explaining when to use FunctionSourceCoverageV2 and when to use the original version. Is V2 potentially a replacement for V1 or an extension to be used in certain places?

Reply:
Fair point. I'll add corresponding comments.

The plan is to deprecate and remove the old one, but I think the old one is used by some external tools (like mutation testing), so wanted to keep it around until it is safe to remove.